qa/rpc-tests/proxy_test.py

```diff
@@ -45,29 +47,36 @@ def __init__(self):
         self.conf2.addr = ('127.0.0.1', 14000 + (os.getpid() % 1000))
         self.conf2.unauth = True
         self.conf2.auth = True
-        # ... one on IPv6 with similar configuration
-        self.conf3 = Socks5Configuration()
-        self.conf3.af = socket.AF_INET6
-        self.conf3.addr = ('::1', 15000 + (os.getpid() % 1000))
-        self.conf3.unauth = True
-        self.conf3.auth = True
+        if self.have_ipv6:
+            # ... one on IPv6 with similar configuration
+            self.conf3 = Socks5Configuration()
+            self.conf3.af = socket.AF_INET6
+            self.conf3.addr = ('::1', 15000 + (os.getpid() % 1000))
+            self.conf3.unauth = True
+            self.conf3.auth = True
+        else:
+            print "Warning: testing without local IPv6 support"
```

Comment:
> Is there any python 3 compatibility at all?

Probably not, but let's not make it worse?

> Adding parenthesis without the proper __future__ import will result in wacky output (in general - in this specific case, there is only one argument so it won't be printed as a string tuple).

How so? There is only ever one argument to print, right?